app/models/miq_report/generator.rb

```diff
@@ -272,17 +268,13 @@ def generate_daily_metric_rollup_results(options = {})
   def generate_interval_metric_results(options = {})
     time_range = Metric::Helper.time_range_from_offset(interval, db_options[:start_offset], db_options[:end_offset])
 
-    # Only build where clause from expression for hourly report. It will not work properly for daily because many values are rolled up from hourly.
-    exp_sql, exp_includes = conditions.to_sql(tz) unless conditions.nil? || db_klass.respond_to?(:instances_are_derived?)
-
     results = db_klass.with_interval_and_time_range(interval, time_range)
                       .where(where_clause)
                       .where(options[:where_clause])
-                      .where(exp_sql)
                       .includes(get_include_for_find)
-                      .includes(exp_includes || [])
```

Comment:
As @jrafanie brought up, my guess is the reason that you brought this up is that you are getting rid of the `.includes()` (possibly?).

Can you provide more clarity at what the rational for this change is?  The commit message is not clear.